subsys/custom_subsys/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+# SPDX-License-Identifier: Apache-2.0
+
+zephyr_interface_library_named(custom_subsys)
```

Comment:
Let us use the same approach that we used in the lib:

```
zephyr_library()
zephyr_library_sources(custom_subsys.c)
zephyr_include_directories(${ZEPHYR_CURRENT_MODULE_DIR}/include/custom_subsys)
```

Reply:
Accepted and manually integrated.